.github/workflows/tag.yml

```diff
@@ -12,14 +12,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install Go
-        uses: actions/setup-go@v1
+        uses: actions/setup-go@v2
         with:
           go-version: 1.14
       - name: Unshallow
         run: git fetch --prune --unshallow
       - name: Login do docker.io
         run: docker login -u golangci -p ${{ secrets.GOLANGCI_LINT_DOCKER_TOKEN }}
       - name: Create release
-        uses: goreleaser/goreleaser-action@v1
+        uses: goreleaser/goreleaser-action@v2
         with:
+          version: latest
           args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GOLANGCI_LINT_TOKEN }}
```

Comment:
nits we can refer to env define in job level as well, something like `${{ env.GITHUB_TOKEN }}`, but I think it's fine to use secrets.GOLANGCI_LINT_TOKEN here :)